web/themes/custom/pineapple/sass/components/navigation/_main-menu.scss

```diff
@@ -1,2 +1,119 @@
+.menu {
+  overflow: hidden;
+  background-color: $primary;
+  text-align: center;
 
+  .hamburger {
+    color: $white;
+    float: left;
+    display: block;
 
+    @include breakpoint($lap) {
+      display: none;
+    }
+  }
+
+  .menu-item {
+    float: left;
+    display: none;
+    color: $white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+
+    @include breakpoint($lap) {
+      display: block;
+    }
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &.is-active {
+      color: white;
+    }
+  }
+
+  .expanded-container {
+    float: left;
+    overflow: hidden;
+
+    &:hover .sub-menu {
+      display: block;
+    }
+
+    .menu-item {
+      font-size: 17px;
+      border: none;
+      outline: none;
+      color: white;
+      padding: 14px 16px;
+      background-color: inherit;
+      font-family: inherit;
+      margin: 0;
+      text-align: center;
+      display: none;
+
+      @include breakpoint($lap) {
+        display: block;
+      }
+    }
+
+    .sub-menu {
+      display: none;
+      position: absolute;
+      background-color: $accent-green;
+      min-width: 160px;
+      box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+      z-index: 1;
+      top: 2.5em;
+
+      .menu-item {
+        float: none;
+        color: $white;
+        padding: 12px 16px;
+        text-decoration: none;
+        display: block;
+        text-align: left;
+
+        &:hover {
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (max-width: $lap) {
+  .menu .hamburger {
+    float: left;
+    display: block;
+  }
+}
+
+@media screen and (max-width: $lap) {
+  .menu.responsive {position: relative;}
+  .menu.responsive .hamburger {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .menu.responsive a {
+    float: none;
+    display: block;
+    text-align: center;
+  }
+  .menu.responsive .expanded-container {
+    float: none;
+  }
+  .menu.responsive .sub-menu {
+    position: relative;
+    top: 0;
+  }
+  .menu.responsive .expanded-container .menu-item {
+    display: block;
+    width: 100%;
+    text-align: center;
+  }
+}
```

Comment:
This code should be integrated with the code above it not separated

Reply:
done